helmfile/charts/sc-logs-retention/scripts/logs-retention.sh

```diff
@@ -49,8 +49,17 @@ else
       BACKUP_URI="s3://${BUCKET_NAME}/logs/${BACKUP}"
 
       echo "Deleting backup ${BACKUP_URI}"
-      aws s3 rm --recursive "${BACKUP_URI}" --endpoint-url="${S3_REGION_ENDPOINT}"
-    elif [[ ${GCS_BACKUP} == "true" ]]; then
+      NR_OF_FILES=$(aws s3 ls "${BACKUP_URI}" --endpoint-url="${S3_REGION_ENDPOINT}" | wc -l)
+      while [ "$NR_OF_FILES" -gt 0 ]; do
+        aws s3 rm --recursive "${BACKUP_URI}" --endpoint-url="${S3_REGION_ENDPOINT}"
+        sleep 5
+        NR_OF_FILES=$(aws s3 ls "${BACKUP_URI}" --endpoint-url="${S3_REGION_ENDPOINT}" | wc -l)
+      done
```

Comment:
Add some comment here that explains why we have this loop (e.g. that it's necessary because not every cloud provider  is implementing the full s3 api), see the issue for inspiration.